.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
```

Comment:
To verify changes, docker build should run on every noticeable change in Pull Request. Example here: https://github.com/qoollo/bob/blob/master/.github/workflows/build.yml#L5